kubernetes-client/src/test/java/io/fabric8/kubernetes/client/internal/KubernetesResourceUtilTest.java

```diff
@@ -220,4 +209,17 @@ void testInferListType() {
     assertEquals(CronJobList.class, KubernetesResourceUtil.inferListType(CronJob.class));
     assertEquals(CustomResourceList.class, KubernetesResourceUtil.inferListType(Good.class));
   }
+
+
+  @Test
+  void testcreateDockerRegistrySecret() throws JsonProcessingException {
+    Secret secret = KubernetesResourceUtil.createDockerRegistrySecret("http://harbor.inner.com", "SecretAdmin", "TestingSecret");
+
+    String header = new String(Base64.getDecoder().decode("eyJhdXRocyI6eyJodHRwOi8vaGFyYm9yLmlubmVyLmNvbSI6eyJwYXNzd29yZCI6IlRlc3RpbmdTZWNyZXQiLCJhdXRoIjoiVTJWamNtVjBRV1J0YVc0NlZHVnpkR2x1WjFObFkzSmxkQT09IiwidXNlcm5hbWUiOiJTZWNyZXRBZG1pbiJ9fX0="));
```

Comment:
You should be getting this value from secret.getData().get(".dockerconfigjson")

Reply:
Yes, Agree. Modified it.